spec/features/users/new_spec.rb

```diff
@@ -1,30 +1,93 @@
 require 'rails_helper'
 
 RSpec.describe 'User Registration', type: :feature do
```

Comment:
Because the test file is nested inside of a features directory we don't have to use type: :feature here.

Reply:
This is stated in the [best practices](https://backend.turing.io/module2/lessons/testing_best_practices) lesson under file structure.

Reply:
New line:
`RSpec.describe 'User Registration' do`

Reply:
thanks I keep forgetting to ask about this